lib/ansible/vars/manager.py

```diff
@@ -474,6 +477,8 @@ def _get_magic_variables(self, play, host, task, include_hostvars, include_deleg
             variables['role_names'] = variables['ansible_play_role_names']
 
             variables['ansible_play_name'] = play.get_name()
+            # get path returns path:line
+            variables['ansible_play_path'] = play.get_path().split(':')[0]
```

Comment:
Should we just call this the `ansible_playbook_path`?  Since we're stripping the line, technically I guess it makes sense to just refer to it as the playbook.

Reply:
with import_playbook, the play will be in diff file from the 'playbook path', why we need 2 diff ones